src/main/kotlin/com/github/biomejs/intellijbiome/BiomeStdinRunner.kt

```diff
@@ -17,54 +18,17 @@ import kotlin.time.Duration
 class BiomeStdinRunner(private val project: Project) : BiomeRunner {
     private val biomePackage = BiomePackage(project)
 
-    override fun format(request: BiomeRunner.Request): BiomeRunner.Response {
-        val commandLine = createCommandLine(request.virtualFile, "format")
+    override fun check(request: BiomeRunner.Request, features: EnumSet<Feature>): BiomeRunner.Response {
+        val commandLine = createCommandLine(request.virtualFile, "check", getCheckFlags(features))
         val file = request.virtualFile
         val timeout = request.timeout
-        val future = startTheFuture(
-            BiomeBundle.message(
-                "biome.failed.to.format.file",
-                file.name
-            ), timeout
+        val failureMessage = BiomeBundle.message(
+            "biome.failted.to.run.biome.check.with.features",
```

Comment:
A tiny typo it should be `"biome.failed.to.run.biome.check.with.features"`

Reply:
Nice catch! Thank you